src/components/activities/patientDetailsActivity/PatientDetailsActivity.tsx

```diff
@@ -63,6 +64,7 @@ const PatientDetailsActivity: FunctionComponent<TProps> = ({
   const { id } = useParams<{ id: string }>();
   const { path, url } = useRouteMatch();
   const history = useHistory();
+  const canEdit = usePermission("patient.update");
```

Comment:
I should check if I can access to this page too (something like If I don't have any permission or if I don't have the read permission to the patient).